### PR TITLE
Fix small mistake and add min / max headers to versioning rfc.

### PR DESCRIPTION
**Pull request by tylercloke**
Ping @chef/lob 

Related: https://github.com/chef/chef-server/pull/233

TODO:
- [ ] Is the `Vary` header still being used as described by the RFC @marcparadise?

### CODE REVIEW

**Comment by marcparadise** [comment]
:+1:

File: rfc041-server-api-versioning.md [hunk 1]
```diff
@@ -178,6 +184,15 @@ Vary: X-Ops-Server-API-Version`
 `$actual_version` MUST indicate the API version level of the response.  The `Vary` header is included
 in the event it becomes desirable to support caching of specific responses.
 
+#### Response Headers
+
+Any request to the server will return the min and max supported API versions as headers in the following format:
```
Comment (danielsdeleo) [inline rfc041-server-api-versioning.md]: You should explain that this is necessary to detect older servers that don't support versioning.

**Comment by danielsdeleo** [comment]
:+1: Aside from my comment

**Comment by sdelano** [comment]
:+1:  after addressing Dan's comment

**Comment by tylercloke** [comment]
@danielsdeleo @sdelano @marcparadise Deleted your thumbsup since I added a bit more.

**Comment by marcparadise** [comment]
:+1: to the revised version

**Comment by stevendanna** [comment]
:+1: on these revisions.

**Comment by ranjib** [comment]
:+1:

**Comment by thommay** [comment]
:+1:

**Comment by stevendanna** [comment]
Does this need to be discussed in the IRC meeting or can @adamhjk decideride on it?  If it needs to get into an IRC meeting, could we get it into todays meeting somehow?

**Comment by jonlives** [comment]
I'd say there are enough additions here that it merits including in the meeting. Can we add it to the agenda please @nathenharvey?

**Comment by thommay** [comment]
It's on the agenda already...

**Comment by jonlives** [comment]
Oh, excellent. As you were then :p

**Pull request merged**